packages/core/src/model.ts

```diff
@@ -355,6 +360,7 @@ export type ParsedViewerConfig = Omit<
     | 'defaultPitch'
     | 'fisheye'
     | 'requestHeaders'
+    | 'renderer'
```

Comment:
This as no use. This type is for when the result of the parser is different than the input.